application/Filters/Honeypot.php

```diff
@@ -1,51 +1,45 @@
-<?php namespace App\Filters;
+<?php
+
+namespace App\Filters;
 
 use CodeIgniter\Filters\FilterInterface;
-use CodeIgniter\HTTP\RequestInterface;
-use CodeIgniter\HTTP\ResponseInterface;
+use CodeIgniter\HTTP\IncomingRequest;
```

Comment:
These should still be the the Interfaces. Specifying an exact class means that if someone replaces IncomingRequest with their own class, this will throw exceptions, which isn't the desired action.

Reply:
ok. I changed it to the descendent classes because of methods called that are not in the interfaces. Will fix :)